kore/src/Kore/Simplify/Ceil.hs

```diff
@@ -261,13 +245,16 @@ newAxiomCeilSimplifier = CeilSimplifier $ \input -> do
             Condition.top
             (synthesize $ CeilF input)
             (const empty)
-    return (OrPattern.map toCondition evaluation)
+    return (OrPattern.map (Predicate.toMultiAnd . toPredicate) evaluation)
   where
-    toCondition Conditional{term = Top_ _, predicate, substitution} =
-        Conditional{term = (), predicate, substitution}
-    toCondition patt =
+    -- TODO(Ana): probably should parse MultiAnd here
+    toPredicate Conditional{term = Top_ _, predicate, substitution} =
+        Predicate.makeAndPredicate
+            predicate
+            (from @_ @(Predicate _) substitution)
+    toPredicate patt =
         error
-            ( "Ceil simplification is expected to result ai a predicate, but"
+            ( "Ceil simplification is expected to result a predicate, but"
```

Comment:
?
```suggestion
            ( "Ceil simplification is expected to result in a predicate, but"
```

Reply:
Sorry, forgot to modify this. I'll do it before merging, don't want to push the change right now because the CI is running.